app/ledger.js

```diff
@@ -981,8 +986,12 @@ var stickyP = (publisher) => {
   var result = (siteSetting) && (siteSetting.get('ledgerPayments'))
 
   // NB: legacy clean-up
-  if ((typeof result === 'undefined') && (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined')) {
-    result = synopsis.publishers[publisher].options.stickyP
+  if (typeof result === 'undefined') {
+    if (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined') {
+      result = synopsis.publishers[publisher].options.stickyP
+    } else {
+      result = true
+    }
     appActions.changeSiteSetting(pattern, 'ledgerPayments', result)
   }
   delete synopsis.publishers[publisher].options.stickyP
```

Comment:
I think you need a check here as well. With those changes `synopsis.publishers[publisher]` is still `undefined` at this point. App is crashing for me and pointing on that line.

Worth note: if you have a site in publisher list error doesn't happen. For publishers on exclusion list (such as Youtube), crash is intermittent. Other publishers instantly crash app.

STR: With a fresh profile (no synopsis, no session, no ledger-state), visit a publisher site (i.e. https://clifton.io)